src/components/mainContent/Navbar.css

```diff
@@ -0,0 +1,40 @@
+.dropdown {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  height: 450px;
+}
+
+.dropdown-content {
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 0 0 10px 10px;
+  display: none;
+  position: absolute;
+  background-color: #f7f7f7;
+  min-width: 160px;
```

Comment:
This number seems a bit arbitrary, is there a reason why you chose it?

Reply:
I don't believe there was a particular reason, this may have been from the CSS on the Figma design. Would this cause future issues?

Reply:
I'm just a bit wary of magic numbers, but we can keep this for now!

Reply:
Since I added a "TODO" for the top/left values, I added a "FIXME" for this one for future reference to make sure that the number doesn't cause any issues/to fix it if it does!